modules/ROOT/pages/bat-bdd-reference.adoc

```diff
@@ -84,6 +84,70 @@ describe `Demo scenario` in [
 
 A test suite must contain at least one test (.dwl) file.
 
+== Keeping Confidential Information Secure in Tests
+
+In your tests, you can access shared secrets in Anypoint Secrets Manager, which allows you to store securely sensitive information, such as a password, authentication token, endpoint URL, webhook URL, or something else. You create aliases for your shared secrets with the `bat grant` command. The BAT CLI stores aliases in your `bat.yaml` files in your test suites. You place aliases in your tests and, at runtime, the BAT CLI looks up the shared secrets by means of the aliases.
+
+[IMPORTANT]
+====
+In a given test suite, you can use aliases for shared secrets only if you plan to run the tests in that suite from a private location.
+====
+
+. Store the information () as a shared secret in Anypoint Secrets Manager.
```

Comment:
I don't understand why " ( ) " is included here.

Reply:
I was typing too exuberantly.